src/serialbox-python/serialbox/type.py

```diff
@@ -80,7 +80,7 @@ def typeID2numpy(typeid):
     """Convert serialbox.TypeID to numpy.dtype
     """
     if typeid == TypeID.Boolean:
-        return np.bool
+        return np.bool_
```

Comment:
```suggestion
        return bool
```
I think that should be the right fix.